src/weaponComponent/InvalidWeaponComponent.ts

```diff
@@ -0,0 +1,32 @@
+import { WeaponComponent } from './WeaponComponent';
+import { WeaponComponentHash } from './WeaponComponentHash';
+import { WeaponHash } from '../hashes';
+import { ComponentAttachmentPoint } from './ComponentAttachmentPoint';
+import { Game } from '../Game';
+
+export class InvalidWeaponComponent extends WeaponComponent{
+  constructor() {
+    super(null, null, WeaponComponentHash.Invalid);
+  }
+
+  public get Active(): boolean {
+    return false;
+  }
+
+  // eslint-disable-next-line @typescript-eslint/no-empty-function
+  public set Active(value: boolean) {
+  }
+
+  public get DisplayName(): string {
+    return 'WCT_INVALID';
+  }
+
+  public get LocalizedName(): string {
+    return Game.GetGXTEntry(this.DisplayName);
+  }
+
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  public static getAttachmentPoint(hash: WeaponHash, componentHash: WeaponComponentHash): ComponentAttachmentPoint{
```

Comment:
Hm, little code consistency issue with no space after ComponentAttachmentPoint. Perhaps just missing an `npm run format`? Although this should be checked already. Weird

Reply:
Oops I manually formatted code on each file lol. I wIll do `npm run format` before commit.

Reply:
Great! I will add a precommit-hook and another check inside Circle-CI to force `prettier --check`. I see we currently do not check if formatting was done.

Reply:
@d0p3t `npm run format` convert line ending from CRLF to LF in entire `/src` codebase. Is there something wrong in my repository? Or should this also be done in this PR lol?

Convert \r\n -> \n
![image](https://user-images.githubusercontent.com/20377660/110261619-ede23880-7feb-11eb-87b9-ecbc1657585d.png)

Modified 154 files / entire codebase
![image](https://user-images.githubusercontent.com/20377660/110261667-1407d880-7fec-11eb-890d-060a6b5c796d.png)